.github/workflows/ci.yml

```diff
@@ -109,9 +109,12 @@ jobs:
   test-docker:
     needs: test-plan
     name: Docker Tests
-    runs-on: ubuntu-latest
     strategy:
-      matrix: ${{fromJson(needs.test-plan.outputs.matrix)}}
+      fail-fast: false
```

Comment:
Why not fail fast?

Reply:
If that's set to true github will cancel all in-progress and queued jobs in the matrix if any job in the matrix fails.